packages/client/ui/react-ui/src/components/CrossmintCollectionView.test.tsx

```diff
@@ -1,34 +1,40 @@
-import "@testing-library/jest-dom";
-import { render, screen } from "@testing-library/react";
+import { cleanup, render, screen } from "@testing-library/react";
+import { beforeEach, describe, expect, test } from "vitest";
 
 import { CrossmintNFTCollectionView } from "./CrossmintNFTCollectionView";
 
 const wallets = [{ chain: "solana", publicKey: "12345" }];
 
-describe("when only passing mandatory fields", () => {
-    test("should add them to the iframe query params", () => {
-        render(<CrossmintNFTCollectionView wallets={wallets} />);
-        const iframe = screen.getByRole("nft-collection-view");
-        const src = iframe.getAttribute("src");
-        expect(src).toContain("wallets=%5B%7B%22chain%22%3A%22solana%22%2C%22publicKey%22%3A%2212345%22%7D%5D");
-        expect(src).toContain("clientVersion=");
+describe("CrossmintNFTCollectionView", () => {
+    beforeEach(() => {
+        cleanup();
```

Comment:
see comment below (not marking as a blocker just a suggestion and maybe even reduce these calls)

Reply:
fixed